lib/Widgets/Utils.vala

```diff
@@ -290,6 +290,26 @@ public static Gdk.RGBA contrasting_foreground_color (Gdk.RGBA bg_color) {
     return fg_color;
 }
 
+/**
+ * Sets the app's icon theme and stylesheet to elementary defaults. Optionally takes a specific stylesheet accent variant as a string.
+ *
+ * @param accent optional stylesheet accent variant to use, e.g. "blueberry"
+ */
+public void force_elementary_style (string? accent = "blueberry") {
```

Comment:
No need to be nullable here since we are defaulting to blueberry if empty and if someone set it to `null`, we will set the theme to `io.elementary.stylesheet.`.

Reply:
D'oh, you're right. Thanks!